src/SprykerEco/Yves/Computop/CheckoutPage/Process/Steps/ComputopPayNowInitStep.php

```diff
@@ -20,14 +20,7 @@ class ComputopPayNowInitStep extends AbstractBaseStep
      */
     public function requireInput(AbstractTransfer $quoteTransfer): bool
     {
-        if (
-            !$quoteTransfer->getPayment()
-            || $quoteTransfer->getPayment()->getPaymentSelection() !== ComputopConfig::PAYMENT_METHOD_PAY_NOW
-        ) {
-            return false;
-        }
-
-        return true;
+        return $quoteTransfer->getPaymentOrFail()->getPaymentSelection() !== ComputopConfig::PAYMENT_METHOD_PAY_NOW;
```

Comment:
Please make sure that this reflects in the API specification (if needed). The same for other `get*OrFail`.